cinema/models.py

```diff
@@ -5,6 +5,32 @@ class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
+    genres = models.ManyToManyField("Genre", related_name="movies")
+    actors = models.ManyToManyField("Actor", related_name="actors")
```

Comment:
The `related_name` for the `actors` field in the `Movie` model is set to 'actors', which is the same as the model name. This can lead to confusion. Consider using a different name, such as 'movies' or 'movie_actors'.